### PR TITLE
Support more licenses from https://huggingface.co/docs/hub/repositori…

### PR DESCRIPTION
**Pull request by marcenacp**
…es-licenses.

### CODE REVIEW

File: editor/views/metadata.py [hunk 1]
```diff
@@ -3,40 +3,25 @@
...
-    "CDLA-Sharing-1.0",
-    "ODC-ODbL",
-    "CC BY-NC",
-    "CC BY-ND",
-    "CC BY-NC-SA",
-    "CC BY-NC-ND",
-]
-
 
 def render_metadata():
```
Comment (ccl-core) [inline editor/views/metadata.py:13]: Any chance we can add a docstring?

File: editor/views/metadata.py [hunk 2]
```diff
@@ -3,40 +3,25 @@
...
     metadata = st.session_state[Metadata]
-    try:
-        index = licenses.index(metadata.license)
-    except ValueError:
-        index = None
+    index = find_license_index(metadata.license)
```
Comment (ccl-core) [inline editor/views/metadata.py:15]: What happens if the value is None?

**Review by ccl-core — APPROVED** [review]
Thanks!

File: editor/views/metadata.py (hunk 2 above)
Comment (marcenacp) [reply editor/views/metadata.py:15]: `None` is a supported value :)

File: editor/views/metadata.py (hunk 1 above)
Comment (marcenacp) [reply editor/views/metadata.py:13]: Done.

**Pull request merged**